.github/workflows/integration.yml

```diff
@@ -62,18 +63,21 @@ jobs:
             return JSON.stringify(matrix);
 
   # This job is meant to emulate what developers working with the Agoric
-  # platform will experience.
-  # It should be kept in sync with the "getting started" workflow at
-  # https://docs.agoric.com/guides/getting-started/
+  # platform will experience.  It should be kept in sync with the "getting
+  # started" workflow at https://docs.agoric.com/guides/getting-started/
+  #
+  # TODO: This job is currently disabled (see GETTING_STARTED_MATRIX_CLI at the
+  # top of this file).  It would be good to reenable when we have actual working
+  # tests.
   getting-started:
-    needs: pre_check
+    needs:
+      - pre_check
+      - getting-started-matrix
     if: needs.pre_check.outputs.should_run == 'true'
     runs-on: ubuntu-latest
-    continue-on-error: true # know failures of getting-started job
```

Comment:
It was a convenient way to mark this as ignored. I'm curious why the change is needed, my understanding is that `"${{ needs.getting-started.result }}" === "success"` when `continue-on-error` is `true`